daemon/main.go

```diff
@@ -354,6 +355,7 @@ func init() {
 		"version", false, "Print version information")
 	flags.Bool(
 		"pprof", false, "Enable serving the pprof debugging API")
+	flags.BoolVar(&cleanAll, "clean-all", false, "Reset Cilium state")
```

Comment:
I don't get this, before you're stopping the program if the socket was running, but you're adding the flag in the daemon.

I think you would like to add this option to cilium CLI, the daemon should not be running.

Reply:
Fixed, but named it `cleanup` for now. If you have a better name please share.